src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop
+} from "@stencil/core";
+import { UP, DOWN, ENTER, HOME, END, SPACE } from "../../utils/keys";
+import {
+  getElementDir,
+  getElementTheme,
+  getElementProp
+} from "../../utils/dom";
+import { guid } from "../../utils/guid";
+import { chevronLeft16 } from "@esri/calcite-ui-icons";
+import CalciteIcon from "../../utils/CalciteIcon";
+
+@Component({
+  tag: "calcite-accordion-item",
+  styleUrl: "calcite-accordion-item.scss",
+  shadow: true
+})
+export class CalciteAccordionItem {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  @Prop({ reflect: true, mutable: true }) active: boolean = false;
+
+  /** pass a title for the accordion item */
+  @Prop() itemTitle: string;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  //--------------------------------------------------------------------------
+
+  @Event() calciteAccordionItemKeyEvent: EventEmitter;
+  @Event() calciteAccordionItemSelected: EventEmitter;
+  @Event() closeCalciteAccordionItem: EventEmitter;
+  @Event() registerCalciteAccordionItem: EventEmitter;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  //--------------------------------------------------------------------------
+
+  componentDidLoad() {
+    this.itemPosition = this.getItemPosition();
+    this.registerCalciteAccordionItem.emit({
+      item: this.el,
+      position: this.itemPosition
+    });
+  }
+
+  render() {
+    const dir = getElementDir(this.el);
+    const theme = getElementTheme(this.el);
+    const scale = getElementProp(this.el, "scale", "m");
```

Comment:
are we setting all the props from computed styles so that they inherit from their parent?